### ISSUE TITLE
Addons: add documentation for `CustomScript`

### ISSUE THREAD

**Issue by humitos** [readthedocs/readthedocs.org#11775]
We need to document this addon and put some examples on how to solve common issues with this feature.

* Refs: https://github.com/readthedocs/readthedocs.org/pull/11758
* Requires: https://github.com/readthedocs/readthedocs.org/issues/11774
* Requires: https://github.com/readthedocs/ext-theme/issues/527
* Requires: https://github.com/readthedocs/addons/pull/474

**Comment by jeyasimma** [comment]
Please assign it to me, I am trying to add the documentation for CustomScript.

**Comment by ericholscher** [comment]
We actually aren't exposing this to users -- should we, and write docs for it?

**Comment by humitos** [comment]
Yeah. I think it's fine to have small docs for this addon. It's not polished/perfect, but it's useful. We can add the other features mentioned in the description of the issue in the following iteration when more people are using it and requesting for upgrades.
